stopify-continuations/src/callcc/jumper.ts

```diff
@@ -84,47 +83,34 @@ function func(path: NodePath<Labeled<FunctionT>>, state: State): void {
   }
   const restoreLocals = path.node.localVars;
 
-  // We instrument every non-flat function to begin with a *restore block*
-  // that is able to re-construct a saved stack frame. When the function is
-  // invoked in restore mode, its formal arguments are already restored.
-  // The restore block must restore the local variables and deal with
-  // the *arguments* object. The arguments object is a real pain and hurts
-  // performance. So, we avoid restoring it faithfully unless we are explicitly
-  // configured to do so.
-  const restoreBlock: t.Statement[] = [ ];
-  // Restore all local variables. Creates the expression:
-  //     [local0, local1, ... ] = topStack.locals;
-  restoreBlock.push(
+  const frame = t.identifier('$frame');
+
+  const restoreBlock = t.blockStatement([
+    t.variableDeclaration('const', [t.variableDeclarator(frame, popRuntimeStack)]),
```

Comment:
I disagree with this refactoring. Undo it.

Reply:
Why do you disagree with this? It marginally faster and the output code is easier to read.

Reply:
I mean, you can change the output code. But, write the code in this style:

```
const restoreBlock: t.Statement[] = [];
restoreBlock.push(...);
````

Reply:
Oh, sure.

Reply:
Done.